PendingReleaseNotes

```diff
@@ -108,3 +108,19 @@
 * The structured output of ``ceph status`` or ``ceph -s`` is now more
   concise, particularly the `mgrmap` and `monmap` sections, and the
   structure of the `osdmap` section has been cleaned up.
+
+* A health warning is now generated if the average osd heartbeat ping
+  time exceeds a configurable threshold for any of the intervals
+  computed.  The OSD computes 1 minute, 5 minute and 15 minute
+  intervals with average, minimum and maximum values.  New configuration
+  option ``mon_warn_on_slow_ping_ratio`` specifies a percentage of
+  ``osd_heartbeat_grace`` to determine the threshold.  A value of zero
+  disables the warning.  New configuration option
+ ``mon_warn_on_slow_ping_time`` specified in microseconds over-rides the
+  computed value, causes a warning
+  when OSD heartbeat pings take longer than the specified amount.
+  New admin command ``ceph daemon mgr.# dump_osd_network [threshold]`` command will
```

Comment:
might make sense to add a little about the admin commands in the main docs too - maybe in doc/rados/operations/monitoring.rst